shared/desktop/package.js

```diff
@@ -0,0 +1,181 @@
+// @flow
```

Comment:
this file is a move of the old one but cleaned up a little (wrapped in a main function and using some more es6 things)